templatestore/frontend/src/pages/templateScreen.js

```diff
@@ -36,8 +37,10 @@ class TemplateScreen extends Component {
             config: {},
             contextData: '',
             attributes: '',
-            version_alias: '',
-            editable: this.props.editable
+            editable: this.props.editable,
+            showAlert: false,
+            alertMessage: '',
```

Comment:
can we not work it out with one variable here?
(lets say if alertMessage is blank, showAlert is false)